Rlabkey/NAMESPACE

```diff
@@ -10,6 +10,11 @@ export("getSchema")
 export("getFolderPath")
 export("saveResults")
 export("RlabkeyUsersGuide")
+export(labkey.experiment.createData)
+export(labkey.experiment.createMaterial)
+export(labkey.experiment.createRun)
+export(labkey.experiment.saveBatch)
+export(labkey.experiment.SAMPLE_DERIVATION_PROTOCOL)
```

Comment:
running 'R CMD check' got warning that labkey.experiment.SAMPLE_DERIVATION_PROTOCOL is not documented, need to add .Rd file for it.